README.md

```diff
@@ -46,8 +45,9 @@ Using Cog
 ---------
 
 **Compiling** Cog follows the usual procedure for projects which use
-[CMake](http://cmake.org): `cmake . && make` should get you started, if your
-system has the needed [dependencies](#dependencies) installed.
+[Meson](https://mesonbuild.com): `meson build && ninja -C build` should get
```

Comment:
Technically, it is `meson setup builddir/`, not `meson builddir/` -- if the first argument is not recognized as a valid subcommand, we insert "setup" before it.

This runs the risk that the build directory name you use clashes with some meson subcommand added in the future -- this is actually the reason why we didn't use "build" as a subcommand name, funnily enough.

We may or may not, in the future, issue deprecation warnings for implicit "setup", who knows.

Reply:
I have updated the instructions to use `meson setup …`, thanks for the tip 👌🏼